_posts/en/newsletters/2021-03-17-newsletter.md

```diff
@@ -0,0 +1,139 @@
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Rescuing lost LN funding transactions:** LN funding transactions are
+  not safe in the presence of transaction malleability.  Segwit
+  eliminated third-party malleability as a concern for most
+  transactions, <!-- by "most", I mean everything signed SIGHASH_ALL -->
+  but it doesn't address the case where the creator of a transaction
+  mutates its txid themselves, such as by fee-bumping the funding
+  transaction using [Replace-by-Fee][topic rbf] (RBF).  If a txid
+  mutation happens, then the pre-signed refund transaction is not valid,
+  so the user can't get their funds back.  Additionally, the remote node
+  may not automatically see the funding transaction and so may not be
+  able to help the funder get their money back.
+
+    This week, Rusty Russell [posted][russell funding rescue] to the
+    Lightning-Dev mailing list about a quick and experimental feature he
+    implemented in C-Lightning to help a user with this problem recover
+    their funds.  He also describes alternative solutions for related
+    problems as well as the impact of the [proposed][bolts #851] channel
+    dual-funding protocol on this problem.  Christian Decker also
+    [posted][decker funding rescue] a [proposed change][bolts #854] to
+    the LN specification to help facilitate funding recovery efforts.
+    As LN software adds support for funding channels from external
+    wallets (e.g. C-Lightning as described in [Newsletter #51][news51
+    cl2672] and LND in [Newsletter #92][news92 lnd4079]), developers may
+    want to give these sort of failure scenarios more attention.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [HWI 2.0.0RC2][hwi 2.0.0] is a release candidate for the next major
+  version of HWI.  FIXME:harding
+
+- [Rust-Lightning 0.0.13][] FIXME:harding
+
+- [BTCPay Server 1.0.7.0][] FIXME:harding
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21007][] adds a new `-daemonwait` configuration option. It has
+  been possible to run Bitcoin Core as a background daemon process since early
+  versions, by starting the program with the `-daemon` configuration option.
+  The `-daemon` option causes the program to immediately start the daemon process
+  in the background and return control to the user. The new `-daemonwait` option
```

Comment:
I think "return control to the user" is a bit confusing to the audience this is intended for (people who need the behavior of Unix daemons described to them).  Additionally, one use for `-daemonwait` is in startup scripts where there's no user in sight and so control can't be returned.  I think this sentence is fine without this final clause, so I'm just going to drop it.

Reply:
Thanks @harding. I agree with you assessment.